internal/provider/resource_image_repo.go

```diff
@@ -388,15 +388,15 @@ func (r *imageRepoResource) Read(ctx context.Context, req resource.ReadRequest,
 		update := (sc.Source.ValueString() != repo.SyncConfig.Source) ||
 			(sc.Expiration.ValueString() != repo.SyncConfig.Expiration.AsTime().Format(time.RFC3339)) ||
 			(sc.UniqueTags.ValueBool() != repo.SyncConfig.UniqueTags) ||
-			(sc.GracePeriod.ValueBool() != repo.SyncConfig.GracePeriod) ||
-			(sc.SyncAPKs.ValueBool() != repo.SyncConfig.SyncApks)
+			(sc.GracePeriod.ValueBool() != repo.SyncConfig.GracePeriod)
+			// Note: SyncAPKs field is deprecated and ignored
 
 		if update {
 			sc.Source = types.StringValue(repo.SyncConfig.Source)
 			sc.Expiration = types.StringValue(repo.SyncConfig.Expiration.AsTime().Format(time.RFC3339))
 			sc.UniqueTags = types.BoolValue(repo.SyncConfig.UniqueTags)
 			sc.GracePeriod = types.BoolValue(repo.SyncConfig.GracePeriod)
-			sc.SyncAPKs = types.BoolValue(repo.SyncConfig.SyncApks)
+			sc.SyncAPKs = types.BoolValue(false) // deprecated field, always set to false
```

Comment:
bleh, this is problematic since we're returning conflicting data for reads (false here, true elsewhere), but also makes me realize that returning different values may not align with the true state of the world upstream.

I don't know if we can return these as-is without causing some state diffs. Let's try to remove these entirely, so we're not accidentally creating diffs and also not lying about the state of the upstream repository - we shouldn't say this is false if it's really true, and defaulting to true doesn't always mean upstream is always true.